include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -650,16 +650,16 @@ class LAPACKFullMatrix : public TransposeTable<number>
 
   /**
    * Compute eigenvalues and eigenvectors of a real symmetric matrix. Only
-   * eigenvalues in the interval $(lower_bound, upper_bound]$ are computed with
-   * the absolute tolerance abs_accuracy. An approximate eigenvalue is
+   * eigenvalues in the interval $(\rm{lower\_bound}, \rm{upper\_bound}]$ are computed with
+   * the absolute tolerance $\rm abs\_accuracy$. An approximate eigenvalue is
```

Comment:
Aren't the brackets missing around `abs\_accuracy`?

Reply:
apparently not.